autogen/agentchat/groupchat.py

```diff
@@ -20,13 +21,21 @@ class GroupChat:
         When set to True and when a message is a function call suggestion,
         the next speaker will be chosen from an agent which contains the corresponding function name
         in its `function_map`.
+    - speaker_selection_method: the method for selecting the next speaker. Default is "auto".
+        - "auto": the next speaker is selected automatically by LLM.
+        - "manual": the next speaker is selected manually by user input.
+        - "random": the next speaker is selected randomly.
+        - "round_robin": the next speaker is selected in a round robin fashion.
```

Comment:
Perhaps "the next speaker is selected in a round robin fashion." could be clarified, "the next speaker is selected in a round robin fashion, i.e., iterating in the same order as provided in `agents`"